src/main/java/org/aludratest/service/smtp/SmtpInteraction.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016 Hamburg Sud and the contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.aludratest.service.smtp;
+
+import org.aludratest.service.AttachParameter;
+import org.aludratest.service.Interaction;
+
+/**
+ * Implementation of the {@link Interaction} interface for the SmtpService.
+ * @author Volker Bergmann
+ */
+public interface SmtpInteraction extends Interaction {
+
+	public void sendMail(
+			String sender, 
+			String[] recipients, 
+			String subject, 
+			@AttachParameter("Message text") String text,
+			String[] filesToAttach, 
```

Comment:
Good review! I provided the requested changes and added a TODO note that currently the email attachments are not logged.